File: app/LanguageQuiz/LanguageQuiz/Country.m
//
//  Country.m
//  Languini
//
//  Created by Leo Thomas on 11/07/15.
//  Copyright (c) 2015 Coding Da Vinci. All rights reserved.
//

#import "Country.h"
#import "Languoid.h"


@implementation Country

@dynamic name;
@dynamic nameDe;
@dynamic code;
@dynamic latitude;
@dynamic longitude;
@dynamic languiod;

+ (NSString *)entityName{
    return @"Country";
}

@end

File: app/LanguageQuiz/Models/Country.swift
//
//  Country.swift
//  Languini
//
//  Created by Leo Thomas on 20/03/16.
//  Copyright © 2016 Coding Da Vinci. All rights reserved.
//

import Foundation
import MapKit

@objc(Country)
class Country: NSManagedObject {

    static let entityName = "Country"

    var location: CLLocation {
        get {
            return CLLocation(latitude: Double(latitude ?? 0), longitude: Double(longitude ?? 0))
        }
    }
}
